test/pytest/test_torch_compile.py

```diff
@@ -64,6 +64,10 @@ def test_start_torchserve(self):
         assert len(glob.glob("logs/model_log.log")) == 1
         assert len(glob.glob("logs/ts_log.log")) == 1
 
+    @pytest.mark.skipif(
```

Comment:
Does this not work? What's the error you get?

Same comment for all of the failing tests curious what's the problem

Reply:
The curl commands don't work. All the tests which are using curl commands need to be refactored using requests

Reply:
Is this because curl is not installed or not working?

Reply:
curl is installed. curl to localhost doesn't work. Tried various things like --network=host etc. Not sure. We can address this in a different PR